pyproject.toml

```diff
@@ -72,12 +73,24 @@ select = [
 ignore = [
     'E401',   # Multiple imports on one line
     'E402',   # Module level import not at top of file
-    'E501',   # Line too long
     'F541',   # f-string without placeholders
     'PIE808', # range() starting with 0
+    # https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
+    'E501',   # Line too long
```

Comment:
I don't think E501 is needed.

Reply:
I'll remove it in this PR, but the point is that sometimes `ruff check` and `ruff format` have different opinions about the line length.  We can always add it back if there is friction.